test/lib/spender/using/income_type.exs

```diff
@@ -0,0 +1,29 @@
+defmodule Spender.Using.IncomeTypetest do
+  @moduledoc """
+    This module contains tests for the income type changeset
+  """
+  use Spender.DataCase
+  alias Spender.Using.IncomeType
+
+  @valid_income_type %{name: "Business", balance: "1500.50"}
+  @invalid_income_type %{}
+
+  describe "Income_type changeset" do
+    test "Is valid when provided the right attributes" do
+      changeset = IncomeType.changeset(%IncomeType{}, @valid_income_type)
+      assert changeset.valid?
+    end
+
+    test "Is invalis when provided with the wrong attributes" do
```

Comment:
This test has a spelling error in its name

Reply:
Noted.